includes/Classifai/Admin/BulkActions.php

```diff
@@ -283,6 +305,14 @@ public function register_language_processing_row_action( array $actions, \WP_Pos
 				continue;
 			}
 
+			if ( ! in_array( get_post_type(), $feature->get_supported_post_types(), true ) ) {
+				continue;
+			}
+
+			if ( ! in_array( get_post_status(), $feature->get_supported_post_statuses(), true ) ) {
+				continue;
+			}
```

Comment:
I think this should be limited to classification feature only as we don't have post status selection in other features like Excerpt generation, TTS etc...

```suggestion
			if ( Classification::ID === $feature::ID && ! in_array( get_post_status(), $feature->get_supported_post_statuses(), true ) ) {
				continue;
			}
```

Reply:
Fixed [b2cde57](https://github.com/10up/classifai/pull/709/commits/b2cde57e60f8fb1e47e8a9aed2fe132d650a6ac9)